python/lsst/ts/m2gui/controltab/tab_config_view.py

```diff
@@ -33,7 +37,233 @@ class TabConfigView(TabDefault):
         Table's title.
     model : `Model`
         Model class.
+
+    Attributes
+    ----------
+    model : `Model`
+        Model class.
     """
 
     def __init__(self, title, model):
         super().__init__(title, model)
+
+        # Labels
+        self._label_file_configuration = create_label()
+        self._label_file_version = create_label()
+        self._label_file_control_parameters = create_label()
+        self._label_file_lut_parameters = create_label()
+
```

Comment:
You can do much better and spare tens of code line by:
1. storing labels in a hash. where the key would be config item name:
```python
      self.config_parameters = { "file_configuration" : create_label() ..
```
or even initialize from a string providing names of configuration items. Then you can write all methods in the class using for loops. Adding new configuration parameter would be the question of adding single char to array holding names, and (if you don't go for options to construct configuration as a hash map) another line to config.py